src/CalcManager/CalculatorVector.h

```diff
@@ -7,9 +7,9 @@ template <typename TType>
 class CalculatorVector
 {
 public:
-    HRESULT GetAt(_In_opt_ unsigned int index, _Out_ TType *item)
+    int32_t GetAt(_In_opt_ unsigned int index, _Out_ TType *item)
     {
-        HRESULT hr = S_OK;
+        int32_t hr = S_OK;
```

Comment:
As discussed, this PR is mostly goodness, but we're holding off on merging to see if we can use a better substitute for `HRESULT` that still reflects the semantics of the datatype.